app/components/avo/fields/common/multiple_file_viewer_component.rb

```diff
@@ -3,10 +3,11 @@
 class Avo::Fields::Common::MultipleFileViewerComponent < ViewComponent::Base
   include Avo::ApplicationHelper
 
-  def initialize(id:, file:, is_image:, resource:, button_size: :md)
+  def initialize(id:, file:, is_image:, direct_upload:, resource:, button_size: :md)
```

Comment:
We need a default here otherwise we'll get a crash.

```suggestion
  def initialize(id:, file:, is_image:, direct_upload: false, resource:, button_size: :md)
```